lib/app.js

```diff
@@ -25,19 +28,21 @@ exports.getDocuments = getDocuments = function(params, callback) {
   });
 }
 
-var options = {
-  hostname: 'localhost',
-  port: 8984,
-  path: '/CSD/documents.json',
-  agent: false
+exports.createChannelConfigObject = createChannelConfigObject = function(name, callback) {
+  var channelConfig = {};
+  fs.readFile('resources/channelConfig.json', function read(err, data) {
+      if (err) {
+          callback(err);
+      }
+      channelConfig = JSON.parse("" + data);
+      channelConfig.name = name;
+      channelConfig.urlPattern = "^/openinfoman/" +name+ "$";
```

Comment:
This won't work. Have a look at the types of careServiceRequest URLs that we need to support that are listed in this issue https://jembiprojects.jira.com/browse/MHER-56. The requests start with /CSD/ then anything, then the document name, then anything again.